WEATHERAPP/Components/SignUp/SignUp.css

```diff
@@ -0,0 +1,87 @@
+body
+{
+   margin: 0;
+}
+.background
+{
+   height: 100vh;
+   background-image:url("../../Assets/L&D-BG\ 4.png");
+   display: flex;
+   flex-direction: column;
+   justify-content: space-evenly;
+   align-items: center;
+}
+.app-header
+{
+   text-align: center;
+   font:700 72px system-ui;
+}
+.signup-form
+{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-evenly;
+    background-color: rgba(132, 95, 95, 1);
+    width: 320px;
+    height: 370px;
+    border-radius: 5px;
+}
+.input-section
+{
+   display: flex;
+   flex-direction: column;
+}
+.input-label
+{
+   color: rgba(255, 255, 255, 1);
+   font-family: sans-serif;
+   font-weight: 100;
+}
+.input-fields
+{
+   width: 270px;
+   height: 30px;
+   border-radius: 6px;
+}
+.signup-button
+{
+    width: 270px;
+    height: 30px;
+    background-color: rgba(44, 44, 44, 1);
+    color: rgba(255, 255, 255, 1);
+    border-radius:5px ;
```

Comment:
Use consistent spacing in the entire file

Reply:
Resolved